CONTRIBUTING.md

```diff
@@ -128,12 +128,15 @@ There are two sets of tests you can run:
 2. Full integration tests (requires cloud environment setups)
 
 #### Local integration tests
-To get local integration tests running, you'll need to have Redis setup:
+To get local integration tests running, you'll need to have Redis and Docker setup:
```

Comment:
I think this is actually outdated - the command run `make test-python-universal-local` no longer requires Redis or Docker